skops/io/_cli.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import annotations
+
+import argparse
+import logging
+import os
+import pathlib
+import pickle as pkl
+from typing import Optional
+
+from skops.io import dumps, get_untrusted_types
+
+
+def _convert(
+    input_file: os.PathLike, output_dir: pathlib.Path, is_trusted: bool = False
```

Comment:
I think the output should be a file name, not a folder name. If the user wants to convert a bunch of files, for now they can write a bash script to do that.

Please also add a docstring.